include/spatio_temporal_voxel_layer/spatio_temporal_voxel_layer.hpp

```diff
@@ -98,6 +98,7 @@ class SpatioTemporalVoxelLayer : public nav2_costmap_2d::CostmapLayer
 
   // Functions to interact with other layers
   virtual void matchSize(void);
+  virtual bool isClearable() {return false;}
```

Comment:
I think this is true, not false, since this has data that can be meaningfully cleared with `clear costmap` service called